pkg/encoding/kzgEncoder/encoder.go

```diff
@@ -51,17 +55,39 @@ type KzgEncoder struct {
 	FFTPointsT [][]bls.G1Point // transpose of FFTPoints
 }
 
-func NewKzgEncoderGroup(config *KzgConfig) (*KzgEncoderGroup, error) {
+func NewKzgEncoderGroup(config *KzgConfig, isEncoder bool) (*KzgEncoderGroup, error) {
```

Comment:
`isEncoder` is confusing as this method always returns an encoder. 
Why don't we name it `loadG2Points` or something like that? 
Also, does it make sense to specify this in config?